onboarding/src/Components/Dashboard/DashboardPage.js

```diff
@@ -39,90 +33,82 @@ function DashboardPage(props) {
     },
   };
 
+  const sessionEmployees = JSON.parse(sessionStorage.getItem('employees'));
```

Comment:
Data stored in `sessionStorage` - usual `SQLite` files like e.g. `storage.sqlite`, `webappsstore.sqlite` etc.
should be checked because it can be compromised.
Strings will be parsed by React (mostly), but url's not.
Adding and using url verification into `utils` would solve this.
```
function validateURL(url, defaultUrl) {
    var a  = document.createElement('a');
    a.href = url;
    if(a.host && a.host != window.location.host)
        return defaultUrl;

    return url;
}
```
BTW: it could be compromised by e.g. "SQLite DB manager". :roll_eyes: